### PR TITLE
Category-topic model analysis

### PR DESCRIPTION
**Pull request by Starrock45**

### CODE REVIEW

File: sections/discussion/category-topic.tex [hunk 1]
```diff
@@ -0,0 +1,62 @@
+\subsection{Category-topic model}\label{sec:discussion_category_topic}
+Specific observations for the category-topic model can also be made.
+Just at for the author-topic model, the similarity between pairs of categories can be calculated.
```
Comment (rasmus-e-christensen) [inline sections/discussion/category-topic.tex]:
```suggestion
Just as for the author-topic model, the similarity between pairs of categories can be calculated.
```

File: sections/discussion/category-topic.tex [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+\end{table}
+
+Of observations to be made here, the second most similar pair, 'Friii' and 'Debat', is interesting to look at.
+While 'Friii' does not seem to have a theme in the articles written, articles with the 'Debat' (debate) category seem to mostly cover themes that can bring differing opinions and articles with interviews.
+This indicates that the model does not find these deeper thematic differences in articles, or that it finds other patterns that humans do not easily see.
+
+It is also interesting that the 'misc' category is seen twice in the top 10 ranking, even though it is made up of many smaller categories with no connection to each other.
+Though, it is not surprising that this thematically mixed category is quite similar to 'Friii' and 'Debat', which are more thematically wide categories.
+
+It is also clear that some of the topics, the model have learned, fit well with how some categories are used in the dataset.
```
Comment (rasmus-e-christensen) [inline sections/discussion/category-topic.tex]:
```suggestion
It is also clear that some of the topics, the model has learned, fit well with how some categories are used in the dataset.
```

**Review by rasmus-e-christensen — CHANGES_REQUESTED** [review]
enkelte rettelser

File: sections/discussion/category-topic.tex (hunk 2 above)
Comment (MrLongbottom) [reply sections/discussion/category-topic.tex]:
```suggestion
It is also clear that some of the topics that the model have learned, fit well with how some categories are used in the dataset.
```

**Review by MrLongbottom — APPROVED** [review]

File: sections/discussion/category-topic.tex (hunk 2 above)
Comment (Starrock45) [reply sections/discussion/category-topic.tex]:
```suggestion
It is also clear that some of the topics that the model has learned, fit well with how some categories are used in the dataset.
```

**Review by rasmus-e-christensen — APPROVED** [review]

**Pull request merged**